spec/workspace-spec.js

```diff
@@ -2652,6 +2652,26 @@ describe('Workspace', () => {
               });
             });
           });
+          describe('pcre2 enabled', async () => {
+            it('supports lookbehind searches', async () => {
+              const results = [];
+
+              await scan(/(?<!a)aa\b/, {}, result => results.push(result));
+
+              expect(results.length).toBe(1);
+              const { filePath, matches } = results[0];
+              expect(filePath).toBe(atom.project.getDirectories()[0].resolve('a'));
```

Comment:
I think that this should fix the linter error:

```suggestion
              expect(filePath).toBe(
                atom.project.getDirectories()[0].resolve('a')
              );
```

Reply:
My bad - I'm used to local eslint installations. That being said, can't install `babel-eslint` due to that gyp/msbuild.exe error again (on windows it needs to be installed locally), so I'll trust you and commit your lint fix directly.